zeus/config.py

```diff
@@ -368,3 +349,32 @@ def configure_scout(app):
 
     ScoutApm(app)
     instrument_sqlalchemy(db)
+
+
+def configure_sentry(app):
+    from sentry_sdk import init
+    from sentry_sdk.integrations.flask import FlaskIntegration
+
+    release = os.environ.get("BUILD_REVISION") or None
+    if not release:
+        try:
+            import subprocess
+
+            release = str(
+                subprocess.check_output(["git", "describe", "--always"])
```

Comment:
Idk if this is exactly equal to what raven did, but afaict this doesn't run in prod